ansible/roles/mcx/templates/console_config.j2

```diff
@@ -0,0 +1,53 @@
+{
+    "CONSOLE_PORT": {
+{% for line, peer_info in device_console_link[inventory_hostname].iteritems() %}
+{% if line != "ConsolePort" %}
+        "{{ line }}": {
+            "baud_rate": "{{ peer_info["baud_rate"] }}",
+            "flow_control": "0",
+            "remote_device": "{{ peer_info["peerdevice"] }}"
+        }{% if not loop.last %},{% endif %}
+
+{% endif %}
+{% endfor %}
+    },
+    "CONSOLE_SWITCH": {
+        "console_mgmt": {
+            "enabled": "yes"
+        }
+    },
+{{ DEVICE_METADATA.stdout }},
+{% if HwSku == "Nokia-7215" %}
+    "INTERFACE": {
+        "Ethernet47": {},
+        "Ethernet47|{{ device_info[inventory_hostname]["ManagementIp"] }}": {
+            "gwaddr": "{{ device_info[inventory_hostname]["ManagementGw"] }}"
+        }
+    },
+{% endif %}
+{% if HwSku == "Celestica-E1031-T48S4" %}
```

Comment:
How about use `{% elif HwSku == "Celestica-E1031-T48S4" %}` to replace above two lines?